tock/projects/models.py

```diff
@@ -219,7 +231,10 @@ class Project(models.Model):
     )
     project_lead = models.ForeignKey(User, null=True, on_delete=models.CASCADE)
     organization = models.ForeignKey(Organization, blank=True, null=True, on_delete=models.CASCADE)
-
+    exclude_from_billability = models.BooleanField(
```

Comment:
I want to clarify this a little more. I *think* anything non-billable should be excluded. @saraheckert does that seem right to you?

If that's the case, then I'm not sure I understand the need =for the new field and check.

@amymok can you help explain it?

Reply:
I think this should probably read, `check if this project should be excluded from calculations of total hours`.  Because this check is really to make sure we accurately calculate the number of nonbillable hours for a reporting period.

Reply:
From the original #940, I think this explains what "excluded" was intended to mean:

> The target billable hours for the week as:
>
> `(total_hours_billed - hours_billed_to_excluded_projects) * billable_expectation)`

So "excluded" hours is the category to which Out of Office lives in.